ocs_ci/utility/vsphere.py

```diff
@@ -332,7 +332,9 @@ def get_unit_number(self, vm):
                     raise VMMaxDisksReachedException
         return unit_number
 
-    def add_disk(self, vm, size, disk_type="thin", ssd=False):
+    def add_disk(
+        self, vm, size, disk_type="thin", ssd=False, stop_vm=True, start_vm=True
+    ):
```

Comment:
for ssd, its not an option to stop vms, its mandatory. So having parameters of stop_vm and start_vm make things complex and error prone if users choose stop_vm as False and ssd. I suggest to remove those parameters